joomla-framework/utilities
# Refactoring IpHelper

Pull Request for Issue #32

### Summary of Changes

* Add tests verifying the bug. These tests reveal several other issues, see [Drone report](https://ci.joomla.org/joomla-framework/utilities/24).

ATTN: This PR builds on PR #31 

### Testing Instructions

Run the tests (or just check [Drone report](https://ci.joomla.org/joomla-framework/utilities/28)).

### Documentation Changes Required

No
